doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -217,7 +217,7 @@ set with::
 :Description: The default policy for using compression if the per-pool property
               ``compression_mode`` is not set. ``none`` means never use
               compression.  ``passive`` means use compression when
-              `clients hint`_ that data is compressible.  ``aggressive`` means
```

Comment:
This kind of changes the meaning; the intent was to say literally "when clients hint that data is compressible", and then link to the reference on how clients would do that.  Can we not keep that?